packages/shared-components/vite.config.js

```diff
@@ -26,13 +26,14 @@ export default defineConfig({
         rollupOptions: {
             // make sure to externalize deps that shouldn't be bundled
             // into your library
-            external: ["react", "react-dom"],
+            external: ["react", "react-dom", "counterpart"],
             output: {
                 // Provide global variables to use in the UMD build
                 // for externalized deps
                 globals: {
                     "react": "react",
                     "react-dom": "ReactDom",
+                    "counterpart": "counterpart",
```

Comment:
I'm not sure this is right

counterpart isn't an identifier in the global context?

<img width="385" height="55" alt="Image" src="https://github.com/user-attachments/assets/4e9eda36-9880-480b-9378-eb51e38863f3" />

whereas `react` is

<img width="909" height="113" alt="Image" src="https://github.com/user-attachments/assets/b600a379-f7b9-4579-866b-ab1604c6120d" />

So how is this meant to get loaded/passed to shared components?

Reply:
So putting into a peer deps is enough?